tests/access/test_access_debug_logging.py

```diff
@@ -24,37 +24,39 @@ class TestAccessDebugLogging(unittest.TestCase):
     ansi_escape = re.compile(r"\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])")
 
     # ============================================================================
-    # Add Access
+    # Alter Access
```

Comment:
Change to:

```python
# Permit Access
```

Reply:
fixed across the board as well as in group connection admin.